layer/nrlf/core/tests/test_validators.py

```diff
@@ -413,6 +419,7 @@ def test_validate_category_too_many_category():
     }
 
 
+# TODO: Parameterize this test and the one below
```

Comment:
is this going to be a tech debt ?

Reply:
I think I could do it right away, I didn't want to make the diff more complicated with non-related code, same as some comment above.

Reply:
fair enough i dont really mind if its not but if its not there should be some tech debt ticket for it